### DISCUSSION TITLE
Duplicate logs collected by agent

### DISCUSSION THREAD

**Discussion by MohitLolo** [apache/skywalking#10332] [General]
**language：java
oap：9.2.0
agent：javaagent:8.13.0
dependency：apm-toolkit-logback-1.x (version 8.13.0)**
The springboot service uses logback to collect logs from the grpc, and obtains duplicate storage for two times
![image](https://user-images.githubusercontent.com/31984024/216278148-a8f644a1-d54d-4b14-8237-03e2f6ed4f7a.png)
![image](https://user-images.githubusercontent.com/31984024/216278238-eb477f68-db45-4438-96cf-f10fbe4d1d21.png)
I don't know whether it is caused by the client retry caused by the timeout of one of the oap parameters. If so, which parameter is adjusted
Each log data is like the following. In fact, I only have one request
![image](https://user-images.githubusercontent.com/31984024/216280369-5082179c-5fd3-43d1-b189-ed7691489a53.png)

**Comment by wu-sheng** [comment]
It is not worth to establish at most once delivery policy for telemetry data.
It costs too much. 
Don't expect APM like a transaction database.

**Reply by wu-sheng** [reply]
You would not have millions of dollars budget to support this feature. Ask for your infra team.
